compiler/optimizer/VPConstraint.cpp

```diff
@@ -4984,9 +4984,9 @@ bool TR::VPClass::mustBeNotEqual(TR::VPConstraint *other, TR::ValuePropagation *
       return getKnownObject()->getIndex() != other->getKnownObject()->getIndex();
 
    TR::VPClass *otherClass = NULL;
-   if (other)
-      otherClass = other->asClass();
-  if (!_preexistence &&
+   otherClass = other->asClass();
+
+   if (!_preexistence &&
```

Comment:
This change is just adding a space before the `if` to fix indentation. I think git got confused with the change right above it.